Homework_2.2/Homework_2_2/Training.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework_2_2
+{
+    class Training : DescriptableLearning, ICloneable
+    {
+        public Lecture[] Lectures { get; set; } = Array.Empty<Lecture>();
```

Comment:
Per requirements:

> The training should store a set of lectures and practice letures in an internal array

The should be single array for storing both PracticalLessons and Lectures